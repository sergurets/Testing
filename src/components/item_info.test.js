import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ItemInfo from './item_info';
describe('Link', () => {
  it('should render correctly', () => {
    const output = shallow(
      <ItemInfo title="title" release_date="date" genres={['rock']} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

//title={el.title} release_date={el.release_date} genres={el.genres}