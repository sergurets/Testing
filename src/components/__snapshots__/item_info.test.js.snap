// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render correctly 1`] = `
<div
  className=""
>
  <div
    className="info_title"
  >
    <span>
      Item 
      title
    </span>
    <span
      className="release_date"
    >
      date
    </span>
  </div>
  <span
    className="info_genres"
  >
    rock
  </span>
</div>
`;
